examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<p>I think it's <code>DownUnder GeoSolutions</code> not <code>Down Under ...</code></p><p>Not sure if you wanna move this acknowledgment to the top or bottom of the file?</p><p> </p><p> </p><blockquote>To do this we utilize Devito <code>TimeFunction</code> constructed two different ways.</blockquote><p>To do this we utilize two distinct <code>TimeFunction</code>'s</p><p> </p><p> </p><blockquote>Serialization uses <code>TimeFunction(..., save=Buffer(M)</code></blockquote><p>Missing )</p><p> </p><p> </p><blockquote>Return control to python</blockquote><p>Python</p><p> </p><p> </p><blockquote>bindary</blockquote><p>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done, fixed capital Python everywhere in nb
bindary --> binary.